scripts/packageTorClient.js

```diff
@@ -19,8 +19,8 @@ const downloadTorClient = (platform) => {
   const torPath = path.join('build', 'tor-client-updater', 'downloads')
   const torS3Prefix = 'https://s3.us-east-2.amazonaws.com/demo-tor-binaries/'
 
-  const torVersion = '0.3.5.7'
-  const braveVersion = '1'
```

Comment:
Just double-checking that this should be going from 1 to 0...?

Reply:
For every tor version update, we reset the brave version to 0. `braveVersion` is only updated if we release multiple binaries for the same tor version. Let me know if that makes sense.

Reply:
Got it, sounds good!